### PR TITLE
fix(camunda-platform): add conditionalVariableName prop

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/camunda/camunda-modeler/issues/2866

image

Old Properties Panel for reference: 

image

### CODE REVIEW

File: src/provider/camunda-platform/properties/ConditionProps.js [hunk 1]
```diff
@@ -333,6 +339,93 @@ function Resource(props) {
...
+      properties: {
+        variableName: value || ''
+      }
+    });
+  };
+
+  return <TextFieldEntry
+    element={ element }
+    id="conditionVariableName"
+    label={ translate('Variable Name') }
```
Comment (MaxTru) [inline src/provider/camunda-platform/properties/ConditionProps.js:387]: We want with sentence case with our new propPanel

```suggestion
    label={ translate('Variable name') }
```

File: src/provider/camunda-platform/properties/ConditionProps.js [hunk 2]
```diff
@@ -333,6 +339,93 @@ function Resource(props) {
...
+      properties: {
+        variableEvents: value || ''
+      }
+    });
+  };
+
+  return <TextFieldEntry
+    element={ element }
+    id="conditionVariableEvents"
+    label={ translate('Variable Events') }
```
Comment (MaxTru) [inline src/provider/camunda-platform/properties/ConditionProps.js:421]: We want with sentence case with our new propPanel

```suggestion
    label={ translate('Variable events') }
```

File: src/provider/camunda-platform/properties/ConditionProps.js [hunk 3]
```diff
@@ -333,6 +339,93 @@ function Resource(props) {
...
+function VariableEventProps(props) {
+  const entries = [];
+
+  entries.push({
+    id: 'conditionVariableName',
+    component: VariableName,
+    isEdited: isTextFieldEntryEdited
+  });
+
+  entries.push({
```
Comment (MaxTru) [inline src/provider/camunda-platform/properties/ConditionProps.js:351]: IMO not for StartEvents (compare existing implementation)

**Review by MaxTru — CHANGES_REQUESTED** [review]

File: src/provider/camunda-platform/properties/ConditionProps.js (hunk 3 above)
Comment (MaxTru) [reply src/provider/camunda-platform/properties/ConditionProps.js:351]: Should also add a testcase for that

Comment (marstamm) [reply src/provider/camunda-platform/properties/ConditionProps.js]: Good catch with the start event. I made it conditional

https://github.com/bpmn-io/bpmn-js-properties-panel/blob/b623f6d8c79ada663c62ace4939d658dbafc0a4e/src/provider/camunda-platform/properties/ConditionProps.js#L355-L361


And added the ConditionalStartEvent to the test case 
https://github.com/bpmn-io/bpmn-js-properties-panel/blob/b623f6d8c79ada663c62ace4939d658dbafc0a4e/test/spec/provider/camunda-platform/ConditionProps.spec.js#L537-L540

**Review by MaxTru — APPROVED** [review]
:wave:

**Pull request merged**